internal/services/keymanager/helpers.go

```diff
@@ -0,0 +1,52 @@
+package keymanager
+
+import (
+	"fmt"
+	"strings"
+
+	key_manager "github.com/scaleway/scaleway-sdk-go/api/key_manager/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/meta"
+)
+
+func UsageToString(u *key_manager.KeyUsage) string {
```

Comment:
Why is this required when the enum got a .String function?

Reply:
Could this util be moved to the SDK? It seems that it might get useful for other tools as well